internal/core/metadata/operators/addressable/get_test.go

```diff
@@ -156,6 +157,48 @@ func TestPortExecutor(t *testing.T) {
 	}
 }
 
+func TestTopicExecutor(t *testing.T) {
+	tests := []struct {
+		name           string
+		mockDb         AddressLoader
+		expectedResult []contract.Addressable
+		expectedError  bool
+	}{
+		{
+			name:           "Successful database call",
+			mockDb:         createMockAddressLoaderName("GetAddressablesByTopic", nil, SuccessfulDatabaseResult, Topic),
```

Comment:
`createMockAddressLoaderName` is confusing here since we're loading by topic. Should we do something like what you've done in the router tests and name this `createMockAddressLoaderStringArg`?